### PR TITLE
add option :use-gripper in :inverse-kinematics

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l [hunk 1]
```diff
@@ -40,13 +40,14 @@
   (:inverse-kinematics
     (target-coords
       &rest args
-      &key link-list
+      &key link-list (use-gripper-jt nil)
```
Comment (k-okada) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l]: `:use-gripper` may better, because we usually use `:use-torso`

**Comment by pazeshun** [comment]
:+1:

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l (hunk 1 above)
Comment (knorth55) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l]: thank you

**Comment by knorth55** [comment]
please merge this

**Comment by knorth55** [comment]
@pazeshun i changed milestone to 1.0, is this ok?

**Comment by pazeshun** [comment]
OK

**Comment by pazeshun** [comment]
@knorth55 You need this to stow-task branch, don't you?

**Comment by knorth55** [comment]
i will rebase

**Pull request merged**